src/core/agent/services/signifyNotificationService.ts

```diff
@@ -0,0 +1,56 @@
+import { AgentService } from "./agentService";
+import { GenericRecordType, KeriNotification } from "../agent.types";
+import { Notification } from "./credentialService.types";
+import { NotificationRoute } from "../modules/signify/signifyApi.types";
+export class SignifyNotificationService extends AgentService {
```

Comment:
Keep to the same style in the repo - which is `export { SignifyNotificationService }` at the bottom